sql/src/main/java/org/opensearch/sql/sql/domain/SQLQueryRequest.java

```diff
@@ -121,8 +121,11 @@ private boolean isFetchSizeZeroIfPresent() {
   }
 
   private boolean isSupportedFormat() {
-    return Strings.isNullOrEmpty(format) || "jdbc".equalsIgnoreCase(format)
-        || "csv".equalsIgnoreCase(format) || "raw".equalsIgnoreCase(format);
+    return Strings.isNullOrEmpty(format)
```

Comment:
Create enum or static array as per https://github.com/Bit-Quill/opensearch-project-sql/pull/237#discussion_r1122568758

Reply:
Added in c4395d6483a1fa5bd2da060ccd31418ba7906820